Client/Frontend/Widgets/TabsButton.swift

```diff
@@ -141,6 +141,7 @@ class TabsButton: UIButton {
         let currentCount = self.countLabel.text
         let infinity = "\u{221E}"
         let countToBe = (count < 100) ? count.description : infinity
+        countLabel.text = countToBe
```

Comment:
This label's text is set down on [L198](https://github.com/brave/brave-ios/pull/181/files#diff-2232a56906839da06d1d6785995815f9R198), so I think this might require taking another look as to why it's really not working.

I.e. Is the if statement just below not executing? Is `updateTabCountUsingTabManager` not being called? Why is this not working anymore since #131 (Could it be TabMO stuff not restoring fast enough? Race condition?)

I have lots of questions haha 😅

Reply:
The statement below executes but it doesn't update the label somehow. Sorry for the lazy solution will dig into it more today or tomorrow :)